mercury/tests/test_views.py

```diff
@@ -139,6 +153,11 @@ def test_StopWatch_GET(self):
         self.assertEqual(200, response.status_code)
         self.assertTemplateUsed("stopwatch.html")
 
+    def test_Event_GET(self):
+        response = self.client.get(reverse(self.stopwatch_url))
```

Comment:
This is probably a copy and paste error. I think it should be 
response = self.client.get(reverse(self.event_url))